include/odp/api/spec/timer.h

```diff
@@ -70,31 +70,32 @@ typedef enum {
  * Return values of timer set calls.
  */
 typedef enum {
-	/**
-	 * Timer set operation succeeded
-	 */
+	/** Timer set operation succeeded */
 	ODP_TIMER_SUCCESS = 0,
 
-	/**
-	 * Timer set operation failed, expiration too early.
-	 * Either retry with a later expiration time or process the timeout
-	 * immediately. */
-	ODP_TIMER_TOOEARLY = -1,
+	/** Timer set operation failed because expiration time is too near to
+	 *  the current time. */
+	ODP_TIMER_TOO_NEAR = -1,
 
-	/**
-	 * Timer set operation failed, expiration too late.
-	 * Truncate the expiration time against the maximum timeout for the
-	 * timer pool. */
-	ODP_TIMER_TOOLATE = -2,
+	/** Timer set operation failed because expiration time is too far from
+	 *  the current time. */
+	ODP_TIMER_TOO_FAR = -2,
 
-	/**
-	 * Timer set operation failed because no event specified and no event
-	 * present in the timer (timer inactive/expired).
-	 */
-	ODP_TIMER_NOEVENT = -3
+	/** Timer set operation failed */
+	ODP_TIMER_FAIL = -3
 
 } odp_timer_set_t;
 
+/** Timer expiration time is too near to the current time. */
+#define ODP_TIMER_TOOEARLY ODP_TIMER_TOO_NEAR
+
+/** Timer expiration time is too far from the current time. */
+#define ODP_TIMER_TOOLATE  ODP_TIMER_TOO_FAR
+
+/** Timer set operation failed because no event specified and no event
+ *  present in the timer (timer inactive/expired). */
+#define ODP_TIMER_NOEVENT ODP_TIMER_FAIL
+
```

Comment:
Should a comment be added that these defines are for backward compatibility and will be deprecated at some point?